package.json

```diff
@@ -8,7 +8,7 @@
       "path": "dist/rudder-analytics.min.js",
       "import": "{ createComponent }",
       "gzip": true,
-      "limit": "35 kB"
+      "limit": "35.5 kB"
```

Comment:
This change is already done. Can you pull the latest changes?

Reply:
Done